en: add what's new in v1.1 (#428)
pingcap/docs-tidb-operator: en/whats-new-in-v1.1.md

---
title: What's New in v1.1
summary: Learn the new features in TiDB Operator v1.1.
category: introduction
---

# What's New in v1.1

> Based on 1.0, TiDB Operator 1.1 adds many new features, including TiDB 4.0 support, TiKV data encryption, TLS certificate configuration. It also support deploying new components such as TiFlash and TiCDC.

Reviewer: ```suggestion
Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```

> Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.

Reviewer 2: ```suggestion
Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration, etc. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```

> Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration, etc. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.

Reviewer: Latin abbreviation is not recommended in technical documentation. `Including` here means inexhaustive and the list can actually go on.
```suggestion
Based on v1.0, TiDB Operator v1.1 has several new features, including TiDB 4.0 support, TiKV data encryption, and TLS certificate configuration. TiDB Operator v1.1 also supports deploying new components such as TiFlash and TiCDC.
```

> TiDB Operator 1.1 also makes improvements in usability, providing user experience consistent with the Kubernetes native resources.

Reviewer: ```suggestion
TiDB Operator 1.1 also makes improvements in usability, providing the user experience that is consistent with the Kubernetes native resources.
```

## Extensibility

- TidbCluster CR supports deploying and managing the PD Discovery component, which is fully capable of replacing tidb-cluster chart to manage the TiDB cluster.

- TidbCluster CR adds support for Pump, TiFlash, TiCDC, and TiDB Dashboard.

> - Add the [Admission Controller](enable-admission-webhook.md) (optional) to improve the experience of upgrade and scaling and to provide the gated release feature.

Reviewer: ```suggestion
- Add the [Admission Controller](enable-admission-webhook.md) (optional) to improve the user experience of upgrade and scaling, and to provide the gated release feature.
```

> - Add the [Admission Controller](enable-admission-webhook.md) (optional) to improve the user experience of upgrade and scaling, and to provide the gated release feature.

Reviewer 2: ```suggestion
- Add the [Admission Controller](enable-admission-webhook.md) (optional) to improve the user experience of upgrade and scaling, and to provide the canary release feature.
```

- `tidb-scheduler` supports high availability (HA) scheduling at any dimension and scheduler preemption.

- Support using tikv-importer chart to [deploy and manage tikv-importer](restore-data-using-tidb-lightning.md#deploy-tikv-importer).

## Usability

- Add TidbMonitor CR to deploy the cluster monitoring.

- Add TidbInitializer CR to initialize the cluster.

> - Add Backup, BackupSchedule, and Restore CR to back up and restore the cluster, which supports using Amazon S3 and GCS as remote storage.

Reviewer: ```suggestion
- Add Backup, BackupSchedule, and Restore CR to back up and restore the cluster, which supports using Amazon S3 or GCS as the remote storage.
```

- Support gracefully restart a component in the TiDB cluster.

## Security

> - Support configuring TLS certificates between TiDB components and for MySQL client.

Reviewer 2: ```suggestion
- Support configuring TLS certificates for the TiDB cluster components and for MySQL clients.
```

- Support TiKV data encryption.

## Experimental features

> - Add `TidbClusterAutoScaler` to implement [automatic scaling](enable-tidb-cluster-auto-scaling.md). You can enable this feature by turning on the `AutoScaling` switch.

Reviewer: ```suggestion
- Add `TidbClusterAutoScaler` to implement [auto-scaling](enable-tidb-cluster-auto-scaling.md). You can enable this feature by turning on the `AutoScaling` switch.
```

> - Add the optional [Advanced StatefulSet Controller](advanced-statefulset.md), which supports deleting a specified Pod. You can enable this feature by turning on the `AdvancedStatefulSet` switch.

Reviewer 2: ```suggestion
- Add the optional [Advanced StatefulSet Controller](advanced-statefulset.md), which supports deleting a specific Pod. You can enable this feature by turning on the `AdvancedStatefulSet` switch.
```
